src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.LinkedList;
+
+/** Stores information about a user including name and userId. */
+public final class User {
```

Comment:
We can probably make the Student and Tutor class a subclass of the User class now.

Reply:
We can also probably add other common things like bio, pfp, email, etc to this class.

Reply:
Oh true! That would actually be super useful with the update to users being able to be both tutor and student. I'll do this in a new PR